src/style.css

```diff
@@ -0,0 +1,111 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc),
+    url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+/* #region search*/
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)